docs/ref/models/lookups.txt

```diff
@@ -112,57 +121,29 @@ following methods:
     Defines the type of class returned by the ``get_lookup()`` method. It must
     be a :class:`~django.db.models.Field` instance.
 
+.. _transform-reference:
+
 Transform reference
 ~~~~~~~~~~~~~~~~~~~
 
-.. class:: Transform
-
-    A ``Transform`` is a generic class to implement field transformations. A
-    prominent example is ``__year`` that transforms a ``DateField`` into a
-    ``IntegerField``.
-
-    The notation to use a ``Transform`` in an lookup expression is
-    ``<expression>__<transformation>`` (e.g. ``date__year``).
-
-    This class follows the :ref:`Query Expression API <query-expression>`, which
-    implies that you can use ``<expression>__<transform1>__<transform2>``.
+A ``transform`` is a special kind of :ref:`Func() expression
```

Comment:
It looks like here is where you start using double-backtick-quoted transform (above uses were unquoted). I think unquoted is better, because there is no specific class or function named `transform` in the Django codebase.